### ISSUE TITLE
Bring Existing Attribute Table to Front Instead of Opening a New One

### ISSUE THREAD

**Issue by morTaZz** [qgis/QGIS#59801]
### Feature description

**Currently**, when clicking on the attribute table for a layer in QGIS, a new attribute table window opens every time. When working with multiple layers and needing to access each table at full extent, this behavior can make it difficult to navigate between the multiple opened attribute tables.

Instead of opening a new attribute table window each time, QGIS could bring the existing attribute table for the selected layer to the front if it sopen. This would streamline the workflow, reduce clutter, and make it easier for users to manage their work.

Benefits:
- Reduces the number of open windows and improves navigation in multi-layer projects.
- Enhances usability by simplifying access to attribute tables.

**Comment by phidrho** [comment]
There are specific workflows where more than one attribute table is useful for same layer:
https://www.youtube.com/watch?v=czmr1SOvIVQ

But this could be a setting in Options.

**Comment by nk9** [comment]
While it's possible to add a setting, I'd argue that it's unlikely that people will want to do this very often. Instead, I'd argue that the default should be to always open the same window. We can then add a button or menu item when the attribute table is frontmost to "clone" the table window.

**Comment by Antoviscomi** [comment]
@nk9 I totally agree with you, sometimes there are hundreds of opened table not two or ten for a single layer. Sometime it's useful to have twice or more than two,  but never is useful to have two thowsand of the same table opened @phidrho
